app/controllers/manageiq/api/base_controller/blueprints.rb

```diff
@@ -41,12 +41,13 @@ def update_bundle(blueprint, bundle)
 
         def deserialize_bundle(bundle)
           options = {}
-          if bundle["service_catalog"]
-            options[:service_catalog] = ServiceTemplateCatalog.find(parse_id(bundle["service_catalog"], :service_catalogs))
-          end
-          if bundle["service_dialog"]
-            options[:service_dialog] = Dialog.find(parse_id(bundle["service_dialog"], :service_dialogs))
-          end
+          options[:service_catalog] = if bundle["service_catalog"]
```

Comment:
Will this key `:service_catalog` if it's not provided in the JSON body? If so, it might delete the service catalog, which might be surprising to the API user